README.md

```diff
@@ -29,17 +28,16 @@ Put the following script and run it to extract OSM data from a place name and tr
 ```groovy
 // Declaration of the maven repository
 @GrabResolver(name='orbisgis', root='http://nexus-ng.orbisgis.org/repository/orbisgis/')
-@Grab(group='org.orbisgis', module='prepare_data', version='1.0-SNAPSHOT')
+@Grab(group='org.orbisgis.orbisprocess', module='prepare_data', version='1.0-SNAPSHOT')
```

Comment:
Not clear for me. What about if we have two module with the name prepare_data.
What about org.orbisgis.geoclimate